src/shared/components/banner-icon-header.tsx

```diff
@@ -14,12 +14,13 @@ export class BannerIconHeader extends Component<BannerIconHeaderProps, any> {
   render() {
     return (
       <div class="position-relative mb-2">
-        {this.props.banner && <PictrsImage src={this.props.banner} />}
+        {this.props.banner && <PictrsImage src={this.props.banner} alt="" />}
         {this.props.icon && (
           <PictrsImage
             src={this.props.icon}
             iconOverlay
             pushup={!!this.props.banner}
+            alt=""
```

Comment:
What's the preferred way to do this? Maybe just have the alt the same as the src? Or maybe use the post title?

Reply:
The post title could work, but would be redundant: screen readers would just hear the same text repeated. The src definitely wouldn't as it takes a *long* time to read out a URL, and it usually doesn't describe the image in question.

The absolute best way to be accessible would be to have an option for users to submit alt text describing their images... but that would be a bit beyond the scope of this ticket.

Empty alt text is used for "decorative images", for example I used it in a few other places where an icon appears next to text. Since the icon is already described by the text it's considered purely decorative. In this case I used the empty alt text because the banner and icon didn't seem to add much to the page for vision impaired users, so this is a clear way of telling them "just skip this".

Further reading: https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Sweet. I'll re-open an old issue on lemmy for us to add an optional image_description as that is entirely possible for post images at least: https://github.com/LemmyNet/lemmy/issues/1086